CODEOWNERS

```diff
@@ -1 +1,2 @@
-*    @ajardin
+*    @ker0x
```

Comment:
I think we can remove this file completely and use organisation teams instead. 😉